memberships/models.py

```diff
@@ -131,3 +131,17 @@ class Membership(models.Model):
     stripe_subscription_id = models.CharField(max_length=255)
     start_date = models.DateTimeField(default=timezone.now)
     end_date = models.DateTimeField(null=True)
+
+
+class FailedPayment(models.Model):
+    stripe_user_id = models.CharField(max_length=255)
+    stripe_subscription_id = models.CharField(max_length=255)
+    stripe_event_type = models.CharField(max_length=255)
+
+    @staticmethod
+    def create(stripe_user_id, stripe_subscription_id, stripe_event_type):
+        return FailedPayment.objects.create(
+            stripe_user_id=stripe_user_id,
+            stripe_subscription_id=stripe_subscription_id,
+            stripe_event_type=stripe_event_type,
+        )
```

Comment:
```suggestion
```

So this isn't really needed here. The reason why it's in the Member model is because i wanted to ensure a Stripe customer was created for every Member.

There are better ways of achieving what I was doing above that I might look to in the future - primarily by way of events.